packages/cashier/src/modules/deposit-crypto/deposit-crypto.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import { observer, useStore } from '@deriv/stores';
+import { Divider } from '../../components/divider';
+import { PageContainer } from '../../components/page-container';
+import { useCashierStore } from '../../stores/useCashierStores';
+import { DepositCryptoCurrencyDetails, DepositCryptoSideNotes, DepositCryptoWalletAddress } from './components';
+import DepositCryptoSideNoteTryFiatOnRamp from './components/deposit-crypto-side-notes/deposit-crypto-side-note-try-fiat-onramp';
+
+const DepositCrypto: React.FC = observer(() => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+    const { general_store } = useCashierStore();
+    const { setIsDeposit } = general_store;
+
+    useEffect(() => {
+        setIsDeposit(true);
+
+        return () => {
+            setIsDeposit(false);
+        };
+    }, [setIsDeposit]);
+
+    return (
+        <PageContainer
+            // Hide the side note and render it in the page content on mobile to match the design,
+            // Need to talk with the design team to put `DepositCryptoSideNoteTryFiatOnRamp` in the
+            // side notes for consistency and then we can remove unnecessary components from the children.
+            right={is_mobile ? undefined : <DepositCryptoSideNotes />}
+        >
+            <DepositCryptoCurrencyDetails />
+            <DepositCryptoWalletAddress />
+            <Divider />
+            {is_mobile && <DepositCryptoSideNotes />}
```

Comment:
Why can't we combine these two line into one as the condition is same?

Reply:
@hamid-deriv You mean like this? The existing one is simpler and more readable I think 🤔 
```tsx
{is_mobile && (
    <>
        <DepositCryptoSideNotes />
        <Divider />
    </>
)}
```